metadata.yaml

```diff
@@ -46,11 +46,17 @@ requires:
     interface: tracing
     optional: true
     limit: 1
+  hacluster:
+    interface: hacluster
+    limit: 1
+    optional: true
 peers:
   tls:
     interface: tls
   cos:
     interface: cos
+  ha:
+    interface: ha
```

Comment:
Why two interfaces?

Reply:
thought i needed peer relation interface, but turned out i didnt. this was a leftover. good catch! removed in [dd1dacb](https://github.com/canonical/mysql-router-operator/pull/177/commits/dd1dacbe74e85bb3b97e4ad7f3a85f911cfaecf0)